cli/Makefile

```diff
@@ -0,0 +1,12 @@
+SHELL   := /bin/bash
+
+# need to change this to VERSION=`git describe --tags`
```

Comment:
Hard-coded version is probably fine as well, as long as we are clear on how it can be updated.

Reply:
agreed, compile time version set as `git describe --tags` important when you develop library that should be available with `go get` from github.com, in this case it's more easy to keep versioning only in one place: git tag, rather than in both tags and source code, lets leave it as it is now than and decide later?